src/mol-plugin/behavior/dynamic/camera.ts

```diff
@@ -28,6 +29,15 @@ const DefaultFocusLociBindings = {
         Trigger(B.Flag.Secondary, M.create()),
         Trigger(B.Flag.Primary, M.create({ control: true }))
     ], 'Camera center and focus', 'Click element using ${triggers}'),
+    clickResetCameraOnEmpty: Binding([
```

Comment:
I am unsure if these constitute a breaking change. Technically if someone else has overridden the bindings in their implementation of the plugin they would need to add these new bindings or it wouldn't work.

Reply:
This might be a breaking change indeed. Can you maybe have a global `const` with the default value and fallback to it inside the implementation if it's undefined?